Packet++/header/NtpLayer.h

```diff
@@ -0,0 +1,649 @@
+#ifndef PACKETPP_NTP_LAYER
+#define PACKETPP_NTP_LAYER
+
+#include "Logger.h"
+#include "IPv4Layer.h"
+#include "Layer.h"
+
+#include "GeneralUtils.h"
+#include "SystemUtils.h"
+
+#include <math.h>
+#include <stdlib.h>
```

Comment:
I think we can move all of these includes to `NtpLayer.cpp` as they're not used in the header file

Reply:
I already moved them. Probably you checked from outdated version

Reply:
I still see:

```cpp
#include "GeneralUtils.h"
#include "SystemUtils.h"
```

In the header file, but I think they're only used in `NtpLayer.cpp` 🤔

Reply:
Understand now. I thought you comment for math.h